src/session-adapters/publicAdapter.ts

```diff
@@ -9,17 +10,15 @@ export type Adapter = {
     clientId: string
     spaceId: string
     userId: string
-    key: string
-  }) => MaybePromise<string | undefined>
+  }) => MaybePromise<AppSession | undefined>
 
   setItem: (params: {
     req: IncomingMessage
     res: ServerResponse
     clientId: string
     spaceId: string
     userId: string
-    key: string
-    value: string
+    value: AppSession
```

Comment:
```suggestion
    session: AppSession
```

I think you mentioned this in the video, but leaving this comment just in case :)